usage/dist/pom.xml

```diff
@@ -98,9 +132,14 @@
                         <configuration>
                             <appendAssemblyId>true</appendAssemblyId>
                             <descriptors>
-                                <descriptor>src/main/config/build-distribution-archive.xml</descriptor>
+                                <descriptor>src/main/config/build-distribution.xml</descriptor>
                             </descriptors>
                             <finalName>brooklyn-${project.version}</finalName>
+                            <formats>
+                                <format>tar.gz</format>
+                                <format>zip</format>
+                            </formats>
+                            <attach>false</attach>
```

Comment:
Yes, better to `<attach>false</attach>` only for the `dir`.  I've removed `attach-artifact` and it works well -- though it was still archiving twice.  Changing to `<phase>package</phase>` seems to fix that, and runs it after tests.